README.md

```diff
@@ -2,14 +2,17 @@
 
 [![CI Status](https://github.com/gradle-nexus/publish-plugin/workflows/CI/badge.svg)](https://github.com/gradle-nexus/publish-plugin/actions?workflow=CI) [![Gradle Plugin Portal](https://img.shields.io/maven-metadata/v/https/plugins.gradle.org/m2/io.github.gradle-nexus/publish-plugin/maven-metadata.xml.svg?label=Gradle%20Plugin%20Portal)](https://plugins.gradle.org/plugin/io.github.gradle-nexus.publish-plugin)
 
-This Gradle plugin is a turn-key solution for publishing to Nexus. You can use it to publish your artifacts to any Nexus instance (internal or public). It is great for publishing your open source to Sonatype, and then to Maven Central, in a fully automated fashion.
+This Gradle plugin is a turn-key solution for publishing to Nexus. You can use it to publish your artifacts to any Nexus Repository Manager 2.x instance (internal or public). It is great for publishing your open source to Sonatype, and then to Maven Central, in a fully automated fashion.
 
 Vanilla Gradle is great, but it cannot fully automate publications to Nexus. This plugin enables isolation of staging repositories so that you can reliably publish from CI, and each publication uses a brand new, explicitly created staging repo ([more](https://github.com/gradle-nexus/publish-plugin/issues/63)). Moreover, the plugin provides tasks to close and release staging repositories, covering the whole releasing process to Maven Central.
 
 This plugin is intended as a replacement of the [Gradle Nexus Staging Plugin](https://github.com/Codearte/gradle-nexus-staging-plugin/) and [Nexus Publish Plugin](https://github.com/marcphilipp/nexus-publish-plugin) duo. See a dedicated [migration guide](https://github.com/gradle-nexus/publish-plugin/wiki/Migration-from-gradle_nexus_staging-plugin---nexus_publish-plugin-duo).
 
 ## Usage
 
+> [!NOTE]  
+> Nexus Repository Manager 3 is not supported. The staging concept and REST API between 2.x and 3.x totally differ.
```

Comment:
I think this would be confusing to most users. e.g. I've never heard about this before. As a first time "I just want to publish my jar" integrator, seeing a "blah not supported" as the first sentence in "how to use this thing?" section would scare me.

Reply:
What do you propose instead?

Reply:
I would go with an h3 "Compatibility" section, probably right before Troubleshooting.

You could create an h4 for Nexus. And I think we should later add Java, Gradle, Kotlin sections at minimum. Or just a bullet list instead of h4. Depends on how much detail.

Reply:
@sradi What do you think about tweaks proposed by Rob?

Reply:
Sounds good. Maybe h4 or bullet list can be added, when there are more than just Nexus compatibility details added in that section.

Reply:
Nice, I updated it a bit (sorry, couldn't suggest because it was complex), @szpak I think it's good to go, please have a [glance](https://github.com/sradi/publish-plugin/blob/nexus3-docs-improvement/README.md#compatibility).

Reply:
Look good, thanks.

Btw, in the next iteration, we would need to re-think and document also (our intentions for) support for https://github.com/gradle-nexus/publish-plugin/issues/330